api/token.go

```diff
@@ -62,13 +68,18 @@ func (a *API) ResourceOwnerPasswordGrant(ctx context.Context, w http.ResponseWri
 		return oauthError("invalid_grant", "Invalid Password")
 	}
 
+	if cookie != "" && config.Cookie.Enabled {
+		a.setCookieToken(ctx, user, cookie == useSessionCookie, w)
```

Comment:
you're not handling the error return. If you're going to ignore it I'd prefer an explicit drop `_ = a.set....`

Reply:
we should at least log the error